src/middlewares/api.middleware.js

```diff
@@ -38,7 +38,12 @@ const apiMiddleware: Middleware = ({ dispatch, getState }) => {
     dispatch(startNetwork(networkLabel));
     apiUtils
       .request({ method, url: path, data, headers })
-      .then(({ body }) => {
+      .then(({ body, header }) => {
+        if (handleHeaders) {
+          const parsedLink = parse(header.link);
```

Comment:
why is the link logic here. isn't it exclusive to github headers? it should be part of the handleHeaders callback
otherwise this would happen in every apiAction you call with handleHeaders